openedx/adg/lms/applications/views.py

```diff
@@ -431,3 +404,14 @@ def is_precondition_satisfied(self):
             bool: True if user has visited the hub page else False
         """
         return not ApplicationHub.objects.filter(user=self.request.user).exists()
+
+    def post(self, request):
+        """
+        Creates an Application Hub object for the user and redirects to Application Hub.
+
+        Returns:
+            HttpResponse object.
+        """
+        ApplicationHub.objects.create(user=self.request.user)
```

Comment:
Isn't get_or_create method would be better if there is a chance post request more than once?